lab0/INSTRUCTIONS.md

```diff
@@ -1,18 +1,20 @@
 Lab 0: Building your first Nerdpack
 ===================================
 
-The purpose of this lab is to provide hands on experience building on top of the New Relic One platform. Following the instructions in this lab, you will be up and running quickly. First, let's make sure you understand a few key concepts that you'll be working with.
+The purpose of this lab is to provide you a quick, hands on experience building on top of the New Relic One platform. First, let's make sure you understand a few key concepts that you'll be working with.
 * **Launcher**: An entry point into the New Relic One platform that is loaded on the "Apps" page.
-* **Nerdlet**: A visual component that renders inside the New Relic One platform at a well defined extension point. For example, a Launcher can invoke a Nerdlet (as you will see shortly). At their core, Nerdlets are React.js components.
-* **Nerdpack** (aka Package): A collection of artifacts, e.g. Nerdlets, that are "packaged" up and published into the New Relic One platform.
-* **Visualization:** This is another Nerdpack artifact that renders your data in a customized way. Unlike Nerdlets, you can add visualizations to dashboards. For now, you'll focus on Nerdlets, but you'll learn about visualizations in a later lab.
+* **Nerdlet**: A visual component that renders inside the New Relic One platform at a well defined extension point. For example, a launcher can invoke a Nerdlet (as you will see shortly). At their core, Nerdlets are React.js components.
+* **Nerdpack** (aka Package): A collection of Nerdpack items, e.g. Nerdlets, that are "packaged" up and published into the New Relic One platform.
+* **Visualization:** This is another Nerdpack item that renders your data in a customized way. Unlike Nerdlets, you can add visualizations to dashboards. 
 
-In Lab0, you will create a Launcher and then add a Nerdlet to it that will be invoked when the user clicks on the Launcher. After this lab you should understand:
+> **Note:** For now, you'll focus on Nerdlets, but you'll learn about visualizations in a later lab.
```

Comment:
Great job on the use of contractions here! However, we try to use the present tense—even when talking about the near future—wherever possible. The reason for this is because the future tense in English is difficult for many readers who's first language isn't English.

Recommended change:

```
> **Note:** Focus on Nerdlets for now. In a later lab, you learn more about visualizations.
```